scripts/notify-telegram.js

```diff
@@ -0,0 +1,87 @@
+const path = require('path')
+const axios = require('axios')
+const fs = require('fs/promises')
+
+const newToken = require('../src/new-token.json')
+
+const TELEGRAM_BOT_TOKEN = process.env.TELEGRAM_BOT_TOKEN
+const TELEGRAM_CHAT_ID = process.env.TELEGRAM_CHAT_ID
+
+if (!TELEGRAM_BOT_TOKEN || !TELEGRAM_CHAT_ID) {
+  throw new Error('TELEGRAM_BOT_TOKEN or TELEGRAM_CHAT_ID is not set')
+}
+
+async function sendMessage(chatId) {
```

Comment:
i think we can directly pass the value of chatID, to the POST request instead of passing it as an argument